### DISCUSSION TITLE
Get a notification on post or comment edits/updates

### DISCUSSION THREAD

**Discussion by voxvargr** [Docile-Alligator/Infinity-For-Reddit#872] [Duplicate Ideas] [answered]
It could be a simple check every so many minutes and on a per post/comment basis not on by default

Useful for megathreads definitely

**Comment by Wladefant** [comment]
Duplicate reddit link https://www.reddit.com/r/Infinity_For_Reddit/comments/vu6suj/option_for_notifications_for_postscomments_that_i/

Everyone who is interested in this feature please upvote

**Comment by Wladefant** [comment, marked as answer]
@spart0n654 this is a duplicate of #963.
Please add your idea as a comment to #963
